data_steward/cdr_cleaner/cleaning_rules/generate_ext_tables.py

```diff
@@ -0,0 +1,197 @@
+import random
+
+import bq_utils
+import resources
+import constants.bq_utils as bq_consts
+import constants.cdr_cleaner.clean_cdr as cdr_consts
+
+EXT_FIELD_TEMPLATE = [
+    {
+        "type": "integer",
+        "name": "{table}_id",
+        "mode": "nullable",
+        "description": "The {table}_id used in the {table} table."
+    },
+    {
+        "type": "string",
+        "name": "src_id",
+        "mode": "nullable",
+        "description": "The provenance of the data associated with the {table}_id."
+    }]
+
+SITE_MAPPING_FIELDS = [
+    {
+        "type": "string",
+        "name": "hpo_id",
+        "mode": "nullable",
+        "description": "The hpo_id of the hpo site/RDR."
+    },
+    {
+        "type": "string",
+        "name": "src_id",
+        "mode": "nullable",
+        "description": "The masked id of the hpo site/RDR."
+    }]
+
+EXT_TABLE_SUFFIX = '_ext'
+MAPPING_PREFIX = '_mapping_'
+SITE_TABLE_ID = 'site_mappings'
```

Comment:
Please prefix `site_mappings` with underscore as this is an internal table we don't release